app/models/services/cbd_commitment_hash.rb

```diff
@@ -49,4 +68,78 @@ def link_attributes
       url: "https://www.cbd.int/action-agenda/contributions/action?action-id=#{@cbd_hash['_id']}"
     }
   end
+
+  def manager_ids
+    @manager_ids ||= begin
+      actor_data = @cbd_hash.dig('actor', 'types')
+      return [] unless actor_data
+      # Generate a list of ids from out database from the cbd ids
+      actor_data.map { |x| cbd_identifier_to_cfn_id(x['identifier']) }
```

Comment:
Is there a reason for using `x` here instead of e.g. `actor_datum` or whatever a single item within `actor_data` represents?

Reply:
sloppiness